src/actions/actions.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios'
+
+export const getCenter='getCenter'
```

Comment:
constants are usually written in all caps and snake-case.
```suggestion
export const GET_CENTER='GET_CENTER'
```